src/algorithms/crypto/randomness.go

```diff
@@ -16,7 +16,7 @@ import (
 // random beacon at a given epoch and incorporating requisite entropy
 GetRandomness(chain personalization crypto.DomainSeparationTag, randEpoch abi.ChainEpoch, entropy []byte) abi.Randomness {
```

Comment:
This should be renamed into GetBeaconEntry, no?

Reply:
This definitely not. It is no longer a beacon entry at this point. It is formatted randomness, specifically for Fil.

Reply:
What I mean to say by "Entry" is a randomness that could be the same randomness for multiple blocks.

If we say GetRandomness, which randomness is that? Is that from the VRF chain? is it the one we called in the design docs an "ElectionSeed"?

Reply:
This will always be from the beacon.